### ISSUE TITLE
<Plural /> not extracted

### ISSUE THREAD

**Issue by rovansteen** [lingui/js-lingui#382]
**Describe the bug**
When using the `<Plural />` component from `@lingui/macro` the translation is not extracted to the messages file.

**To Reproduce**
Steps to reproduce the behavior, possibly with minimal code sample, e.g:

```jsx
import { Plural } from "@lingui/macro"

export default function App() {
   return <Plural value={pagination.total} one="# Form" other="# Forms" />
}
```

And then run `npx lingui extract`

**Expected behavior**
I expect the messages json file to contain the following line:
` "{0, plural, one {# Form} other {# Forms}}": "{0, plural, one {# Form} other {# Forms}}",`

**Additional context**

- jsLingui version ``lingui --version``: 
```
"@lingui/cli": "^2.7.0",
"@lingui/macro": "^2.7.0",
"@lingui/react": "^2.7.0",
```
- Babel version ``npm list babel-core``
`"@babel/core": "^7.1.0"`
- Your Babel config (e.g. `.babelrc`) or framework you use (Create React App, Meteor, etc.)
Using create-react-app (`react-scripts@2.0.4`)
- lingui configuration
```json
  "lingui": {
    "sourceLocale": "en",
    "format": "minimal",
    "localeDir": "<rootDir>/src/locale",
    "srcPathDirs": [
      "src"
    ]
  },
```

Another odd thing is that if I add an empty `<Trans />` component somewhere in the file it does work. So the following code *does* correctly extract the plural translation.
```jsx
import { Plural, Trans } from "@lingui/macro"

export default function App() {
 return (
    <>
      <Trans />
      <Plural value={pagination.total} one="# Form" other="# Forms" />
    </>
  );
}
```

**Comment by ghostd** [comment]
Hi,

Maybe a duplicate of #372 ?

**Comment by rovansteen** [comment]
@ghostd don't think it's a duplicate, as in my case the plural translation is not extracted at all. Could be related of course, thanks for mentioning!

**Comment by revinguillen** [comment]
FWIW, it looks like doing `import { i18nMark } from '@lingui/react';` is a sufficient substitute for importing `Trans`; I have a file with a bunch of `i18nMark`d strings, and one `<Plural>`, and if I remove the former, the latter no longer extracts properly.

**Comment by tricoder42** [comment]
This indeed looks like a bug. Extract plugin checks the imports and macro adds them. There must be some misconfiguration fo both. I'll check over the weekend.

**Comment by revinguillen** [comment]
Fantastic, thank you so much; I look forward to using this in production 👍

**Comment by tricoder42** [comment]
Released in [v2.7.1](https://github.com/lingui/js-lingui/releases/tag/v2.7.1)
